README.md

```diff
@@ -59,14 +59,62 @@ cp -i config/example_config.yml ~/.dp-cli-config.yml
 vi ~/.dp-cli-config.yml
 ```
 
+[ set paths for:
+
+    dp-setup-path:
+    dp-hierarchy-builder-path:
+    dp-code-list-scripts-path:
+  
+   set your `ssh-user:`
+
+and if this is a first time setup, comment out production from environments, thus:
+
+    # - name: production
+    #   profile: production
+]
+
 ### Build and run
 
+Check you have a suitable version of `go` installed with:
+
+`go version`
+
+(Ideally 1.13)
+
+[ The following will ensure version 1.13
+
+  ```sh
+  brew install go@1.13
+  brew unlink go
+  brew link —force go@1.13
+  ```
+
+Check desired version of `go` is on your PATH with `echo $PATH` and if not, either edit your .zshrc file to have the correct path OR do:
+
+  ```sh
+  echo 'export GOPATH=$HOME/go' >> ~/.zshrc
+  echo 'export PATH="/usr/local/opt/go@1.13/bin:$PATH"' >> ~/.zshrc
+  ```
+
+  and restart the terminal ]
+
```

Comment:
These are good instructions, but I'd put them in the "prerequisites" section earlier in the file, perhaps with a reminder link here. Also the specifics of installing go might be better as a link to the `dp` repo's instructions for installing go so we've got in one place and we aren't duplicating.

Reply:
Hi Jess,
I've moved 'go' install to prerequisites.
However, the link in:
https://github.com/ONSdigital/dp/blob/master/guides/INSTALLING.md#prerequisites
about installing go1.13 takes you to the go web site for installing the lates version of go1.15.x which i'm told is not a version to move to yet ...
So, if anything, the link in :
https://github.com/ONSdigital/dp/blob/master/guides/INSTALLING.md#prerequisites
should be updated to reference the fact that go1.13 should already have been installed as per the instructions in this doc.
Something like that ...